### PR TITLE
11788_Migration, Init Seeder

### PR DESCRIPTION
**Pull request by tuananhp-1844**
https://edu-redmine.sun-asterisk.vn/issues/11788

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -25,7 +27,8 @@ class User extends Authenticatable
      * @var array
      */
     protected $hidden = [
-        'password', 'remember_token',
+        'password',
```
Comment (framgiaciv3) [inline app/Models/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Models/User.php</br>**Line**: 30</br>Whitespace found at end of line

**Comment by tuananhp-1844** [comment]
ready

**Comment by tuananhp-1844** [comment]
ready

**Comment by tuananhp-1844** [comment]
ready

**Comment by tuananhp-1844** [comment]
ready

**Pull request merged**